libs/etterlatte-vedtaksvurdering-model/src/main/kotlin/VedtakDto.kt

```diff
@@ -75,11 +74,12 @@ enum class VedtakStatus {
     IVERKSATT,
 }
 
-data class Behandling(
-    val type: BehandlingType,
+// TODO: denne burde bare bli hentet rett fra Behandling...
+data class ForenkletRevurderingInfo(
```

Comment:
Er ikke dette bare en behandling, hva er grunnen til at vi ønsker å endre navn? Er det fordi man forveksler den med "ekstern" behandling? 

Denne vil jo inneholde både førstegangsbehandlinger og revurderinger. Derfor de to ekstra parameterene for revurderinger når det er aktuelt.

Reply:
Endret til "ForenkletBehandling"